vision_datasets/common/constants.py

```diff
@@ -6,7 +6,8 @@ class DatasetTypes:
     IMCAP = 'image_caption'
     IMAGE_TEXT_MATCHING = 'image_text_matching'
     IMAGE_MATTING = 'image_matting'
-    VALID_TYPES = [IC_MULTILABEL, IC_MULTICLASS, OD, MULTITASK, IMCAP, IMAGE_TEXT_MATCHING, IMAGE_MATTING]
+    REGRESSION = 'regression'
+    VALID_TYPES = [IC_MULTILABEL, IC_MULTICLASS, OD, MULTITASK, IMCAP, IMAGE_TEXT_MATCHING, IMAGE_MATTING, REGRESSION]
```

Comment:
As we have text data now, can we make it IMAGE_REGRESSION = 'image_regression' to be more clear? Thanks

Reply:
Changed globally. Thanks!